README.md

```diff
@@ -254,7 +254,7 @@ jobs:
       - run: npm test
 
       - run: npm run release
-        with:
+        env:
```

Comment:
is this change intentional?

Reply:
no, I thought you made that one but I'm switching it back